examples/kubernetes-config/kubernetes-cloud-controller-manager.json

```diff
@@ -0,0 +1,39 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "orchestratorRelease": "1.8",
+      "kubernetesConfig": {
+        "useCloudControllerManager": true

```

Comment:
Might be beneficial to put a CloudControllerManagerConfig field here for reference

Reply:
I'm not sure I want to encourage someone to use this yet until we spend some time testing various custom configurations that actually work! :)